sphinx_jupyterbook_latex/transforms.py

```diff
@@ -297,9 +297,8 @@ class LatexToctreeNodeInterpret(SphinxTransform):
 
     def apply(self, **kwargs: Any) -> None:
         for tocnode in self.document.traverse(toctree_node):
-            if tocnode.attributes["hidden"]:
+            if "hidden" in tocnode.attributes and tocnode.attributes["hidden"]:
```

Comment:
another way to write this would be:
```suggestion
            if tocnode.attributes.get("hidden"):
```